build/ferretdb/evaluation.Dockerfile

```diff
@@ -116,6 +116,10 @@ EOF
 COPY --from=evaluation-build /src/bin/ferretdb /usr/local/bin/ferretdb
 COPY build/ferretdb/99-start-ferretdb.sh /docker-entrypoint-initdb.d/
 
+RUN mkdir -p /var/log/supervisor
+COPY --from=evaluation-build /src/build/ferretdb/evaluation/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/conf.d/supervisord.conf"]
```

Comment:
https://docs.docker.com/engine/containers/multi-service_container/#use-a-process-manager

Reply:
supervisord is a runit alternative written in Python. Not sure why we want to use it.

Reply:
testing graceful shutdown, it seems I can't quite get runit to work with graceful shutdown. metrics and logs were not flushed (and probably pg wasn't shutdown) before the docker container exited :thinking: